setup.py

```diff
@@ -145,6 +145,7 @@ def run_setup(with_extensions):
             "License :: OSI Approved :: BSD License",
         ],
         keywords='smb smb-testing smb-client',
+        options={"bdist_wheel": {"universal": True}},
```

Comment:
same comment here, what is the perceived advantage of building a universal wheel when this change includes provisions for building platform specific wheels?

i think what we really need is a universal wheel for the pure-python package that's uploaded to PyPI and then platform wheels for the compiled extension packages for artifactory.

Reply:
Removed the `universal` option now.